modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -261,6 +261,15 @@ definition {
 		return ${objectDefinitionClassName};
 	}
 
+	@summary = "Default summary"
+	macro _getObjectDefinitionClassNameSuffix(objectDefinitionId = null) {
+		var objectDefinitionClassName = ObjectDefinitionAPI._getObjectDefinitionClassNameById(objectDefinitionId = ${objectDefinitionId});
+
+		var objectDefinitionClassNameSuffix = StringUtil.extractLast(${objectDefinitionClassName}, "#");
+
+		return ${objectDefinitionClassNameSuffix};
+	}
```

Comment:
@vngoliveira could you please change to be `_getObjectDefinitionPortletId` returning:

```
return "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectDefinitionClassNameSuffix}";
```
?